MinPlatformPkg/Test/Library/TestPointCheckLib/DxeTestPointCheckLib.c

```diff
@@ -1265,6 +1271,43 @@ TestPointExitBootServices (
   return EFI_SUCCESS;
 }
 
+/**
+ Test that required devices have trained to the required link speed.
+
+ @retval EFI_SUCCESS            Test was performed and flagged as verified or error logged.
+**/
+EFI_STATUS
+EFIAPI
+TestPointPciEnumerationDonePcieGenSpeed ()
+{
+  EFI_STATUS  Status;
+  BOOLEAN     Result;
```

Comment:
Not a big deal but `Result` seems a bit unnecessary and complicates the logic. Can't an `if` statement based on the return status from `TestPointCheckPciSpeed()` work?

Reply:
Probably? I was making my code blend in with the structures used by the other TestPoints, and assuming they had good reason to be written that way. Perhaps in some unlikely future `EFI_ERROR()` wound be entirely overhauled. I was going to leave it like that for consistency.

Reply:
Okay, I think that was written for a place where it made sense and the got copied around. Like I said, not harmful though.